test/CMakeLists.txt

```diff
@@ -42,7 +42,22 @@ add_subdirectory(
 # Unit Tests
 ##############
 enable_testing()
-file(GLOB TEST_SRC_FILES "${CMAKE_SOURCE_DIR}/test/*.cpp")
+set(TEST_SRC_FILES
+    "${CMAKE_SOURCE_DIR}/test/test_actuators.cpp"
+)
+#[[
```

Comment:
What is that for?

Reply:
It was before you showed to me that it was possible to select which test run or not.
This change in CMakeLists.txt was another way (less good) allows me to not run the others test_....cpp in order to debug, I removed that now.